src/guardian/guardian.service.spec.ts

```diff
@@ -104,7 +103,7 @@ describe('GuardianService', () => {
     }));
 
     const getBlockGuardServiceMock = jest
-      .spyOn(blockGuardService, 'isNeedToProcessNewState')
+      .spyOn(guardianService, 'isNeedToProcessNewState')
```

Comment:
'blockGuardService' is defined but never used

Reply:
changed in fix/val-1216-gurdian-service-impr